src/test/java/io/confluent/connect/jdbc/dialect/BaseDialectTest.java

```diff
@@ -469,19 +480,25 @@ public void bindFieldNull() throws SQLException {
   @Test(expected = ConnectException.class)
   public void bindFieldStructUnsupported() throws SQLException {
     Schema structSchema = SchemaBuilder.struct().field("test", Schema.BOOLEAN_SCHEMA).build();
-    dialect.bindField(mock(PreparedStatement.class), 1, structSchema, new Struct(structSchema));
+    ColumnDefinition colDef = mock(ColumnDefinition.class);
```

Comment:
It seems like you mock this each time. Any chance you can pull it out to lessen duplication of code?